website/docs/components/form/input-group/partials/guidelines/overview.md

```diff
@@ -0,0 +1 @@
+An Input Group combines one or more input fields, buttons, toggles, and other form controls into a single component to handle multi-directional data, search, and fine-grained filtering within an object or dataset.
```

Comment:
I find it difficult to process what 'multi-directional data' refers to – would you be able to expand on this a bit?